.github/workflows/deploy.yml

```
@@ -101,23 +101,26 @@ jobs:
       SLACK_COMMIT_URL: https://github.com/go-fjords/${{ inputs.github-repository }}/commit/${{ inputs.commit-sha }}
     steps:
       - name: Checkout
-        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # ratchet:actions/checkout@v3
+        uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # ratchet:actions/checkout@v3
         with:
           ref: ${{ inputs.commit-sha }}
       - name: Generate commit message summary
         id: generate-commit-summary
         run: |
-          commitSummary=$(git log -1 --pretty=%B | head -n 1 | jq -Rsa .)
-          commitAuthor=$(git log -1 --pretty=%an)
-          echo "::set-output name=commitSummary::$commitSummary"
-          echo "::set-output name=commitAuthor::$commitAuthor"
+          echo 'commitSummary<<xEOFx' >> $GITHUB_OUTPUT
+          git log -1 --pretty=%B | head -n 1 | jq -Rsa . >> $GITHUB_OUTPUT
+          echo 'xEOFx' >> $GITHUB_OUTPUT
+
+          echo 'commitAuthor<<xEOFx' >> $GITHUB_OUTPUT
+          git log -1 --pretty=%an >> $GITHUB_OUTPUT
+          echo 'xEOFx' >> $GITHUB_OUTPUT
```

Is there a reason for the xEOFx? I've not seen the syntax before 😅

```suggestion
          commitSummary=$(git log -1 --pretty=%B | head -n 1 | jq -Rsa .)
          commitAuthor=$(git log -1 --pretty=%an)
          echo "commitSummary=$commitSummary"
          echo "commitAuthor=$commitAuthor"
```

---

To support multiline into the output vars, it wont work like the old syntax did. The 'xEOFx' itself is just a marker, it could be anything (that is unlikely to be found in the input)